config/semi_automated.yml

```diff
@@ -461,6 +461,9 @@ parameters:
             regex: '\(\[\[User:Galobtter\/Shortdesc helper\|Shortdesc helper\]\]\)'
             link: User:Galobtter/Shortdesc helper
             namespaces: [0]
+        Undo-last-edit:
+            regex; '\(\[\[User:Enterprisey\/undo-last-edit\.js\|undo-last-edit\]\]\)'
+            link: User:Enterprisey/undo-last-edit
```

Comment:
This is missing a label. AutoEdits should probably fall back to using the key for the tool (`undo-last-edit`) if a label is missing, but for now it needs to be added explicitly. So all we need is `label: undo-last-edit`

Reply:
Line 12 says that "label: Text to use as the label, defaults to the tool's key. Use this for translations." and most of the other tools don't have labels - is this needed?

Reply:
@DannyS712 Bah you're right. I was going off of the test failure. Turns out the culprit is `regex;`, which should be `regex:` :)

Also, since this is revert-only, let's add `revert: true`

Reply:
Done